Makefile

```diff
@@ -52,8 +52,16 @@ html-lint: ## Validates HTML in web application template files.
 	html_lint.py --printfilename --disable=optional_tag,extra_whitespace,indentation \
 		securedrop/source_templates/*.html securedrop/journalist_templates/*.html
 
+.PHONY: yamllint
+yamllint: ## Lints YAML files (does not validate syntax!)
+# Prune the `.venv/` dir if it exists, since it contains pip-installed files
+# and is not subject to our linting.
+	find "$(PWD)" -path "$(PWD)/.venv" -prune \
+		-o -type f -regextype posix-extended -iregex '^.*\.ya?ml$$' \
+		-exec yamllint -c "$(PWD)/.yamllint" {} +
```

Comment:
Bad news: on macOS `-regextype posix-extended` does not work (it's not a recognized option), but replacing `-regextype posix-extended` with `find -E` works: 

```
.PHONY: yamllint
yamllint: ## Lints YAML files (does not validate syntax!)
# Prune the `.venv/` dir if it exists, since it contains pip-installed files
# and is not subject to our linting.
        find -E "$(PWD)" -path "$(PWD)/.venv" -prune \
                -o -type f -iregex '^.*\.ya?ml$$' \
                -exec yamllint -c "$(PWD)/.yamllint" {} +
```

Reply:
Thanks, I was worried about that. Right now we only have `.yml` files, so what I'll do instead is only lint those (which doesn't require usage of `-regextype`), then in a second find task, confirm that the number of files ending in `.yaml` is zero. That will still cover both bases in CI.

Reply:
@redshiftzero I went with a grep-find-xargs approach, which I hope works equally well on macOS. Let me know if not, and we can pair on a better solution!